app/views/users/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<h1>Form</h1>
+
+<%= form_for(@user) do |f| %>
+  <% if @user.errors.any? %>
+        <% @user.errors.full_messages.each do |msg| %>
+            <div class="alert alert-danger"><%= msg %></div>
+        <% end %>
+  <% end %>
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+  <br>
+  <%= f.label :email %>
+  <%= f.email_field :email %>
+  <br>
+  <%= f.label :password %>
+  <%= f.password_field :password %>
+  <br>
+  <%= f.submit "Create new user" %>
+<% end %>
+
+<%# form_tag form %>
+<%# <%= form_tag('/posts', method: :post, authenticity_token: form_authenticity_token ) do -%>                                                                            
```

Comment:
You just need to pass a route helper to the `form_tag` method, please, check what is the helper you need in this case. Also, see error #2 when I try to create the user.

Reply:
Updated on next commit!